RealEstate/apps/core/static/style.css

```diff
@@ -0,0 +1,110 @@
+/* common */
+
+ul.list-group {
+	padding-top:30px;
+}
+#heading {
+	padding-bottom: 30px;
+}
+.icon {
+	display:none;
+	padding-left: 20px;
+	font-size: 90%;
+}
+.ok {
+	color: green;
+}
+.delete {
+    color: #A94431;
+    cursor: pointer;
+}
+.edit {
+    color: #31708f;
+    cursor: pointer;
+}
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+.inner-rating {
+	padding:20px 0;
+	padding-left:10%
+}
+.slider-selection {
+	background: #BABABA;
+}
+#helptext-L{
+	padding-right: 15px;
+}
+#helptext-R{
+	padding-left: 15px;
+}
+button{
+	margin:30px;
+}
+.modalHeight{
+	height:auto;
+}
+.row-con{
+   display:table;
+   width:100%;
+}
+.hbdash-center {
+    display: table-cell;
+    vertical-align: middle;
+    text-align:center;
+    width:50%;
+}
+
+/* categories */
+
+.rating-col { 
+	min-width: 500px; 
+}
+.alert-success {
+	margin-top: 30px;
+}
+.summary-col{ 
+	min-width: 150px; 
+}
+.tooltip-main.top.in { 
+	z-index:1 !important; 
+}
+.close { 
+	z-index: 3000 !important; 
+}
+#catRating-out{
+	padding-left:10%
+}
+
+/*realtorHome */
+
+.mail {
+    color: #cccccc;
+}
```

Comment:
If this is the class for the span, it needs to be 
color: #cccccc;
display: none;